### PR TITLE
Issue #41 : Real estate optimization

### PR DESCRIPTION
**Pull request by digi0ps**
Changes:
- Removed navbar from *bot-web-view.ejs*
- Title is now rendered using d3
- The y-pos of each svg element is determined using the clientRect coordinates of the previous elements
- Reduced the padding of *#timeline-container*

### CODE REVIEW

File: public/js/app/utils/bargraph.js [hunk 1]
```diff
@@ -91,7 +96,7 @@ define(["d3", "twemoji"], function(d3) {
                 //Tweet Count
                 var barTweetCount = barGroup.append("g")
                     .attr("class", "bar-tweet")
-                    .attr("transform", "translate(" + (width / 2 - 40) + ",50)");
+                    .attr("transform", "translate(" + (width / 2 - 40) + ","+ (yPos+55) + ")");
```
Comment (prastut) [inline public/js/app/utils/bargraph.js:99]: Does it work for all screen sizes?

File: public/js/app/utils/events.js [hunk 2]
```diff
@@ -38,6 +38,11 @@ define(["d3", "twemoji"], function(d3) {
 
         function chart(selection) {
 
+            //Bounding rectangle of the scatterChart
+            var scatterBox = document.getElementById('scatter-g').getBoundingClientRect();
+            yPos = scatterBox.top + scatterBox.height - 20;
```
Comment (prastut) [inline public/js/app/utils/events.js]: Same with this. The factor of -20 is a hardcoded value.

**Review by prastut — COMMENTED** [review]
For all screen sizes is it working? Preferably 15' or Nexus 5/Iphone 6 displays.

**Comment by prastut** [comment]
@digi0ps have a screenshot in the PR as well. Helps in having an overall view of the things that have been changed. I don't need to pull the PR down to check again if there is a screenshot. 

Preferably the screenshot should be when the screen display is of iPhone or Nexus using the Chrome Dev tools.

File: public/js/app/utils/bargraph.js (hunk 1 above)
Comment (digi0ps) [reply public/js/app/utils/bargraph.js:99]: Yes this does. 50 is just the height of the bar chart + 5 for margin.

**Comment by digi0ps** [comment]
image

The -20 was to reduce the free space between scatter chart and events chart. But looks fine without it. So I have removed it.

image

**Comment by prastut** [comment]
This is perfect. 👏 👏

**Pull request merged**